### ISSUE TITLE
Can i turn off  marker "hover to enlarge preview"

### ISSUE THREAD

**Issue by microdockman** [stashapp/stash#4088]
The pc device with mouse operation experience is good, but on the mobile device with finger operation is very troublesome, will constantly enlarge the preview of a certain video

![Snipaste_2023-09-05_10-21-45](https://github.com/stashapp/stash/assets/31898637/3a6e517d-cd5e-4aa7-978b-10e7679bea38)

**Comment by MrX292** [comment]
Is your browser set to desktop mode?

**Comment by microdockman** [comment]
> Is your browser set to desktop mode?

No, I didn't change the browser Settings, everything is the default, and the hover preview works on both the pc and the ipad, except that manually swiping the page on the ipad always triggers the hover display, affecting the ability to swipe up and down

The browser is chrome

**Comment by MrX292** [comment]
Was the screenshot from the iPad or pc?
At least for phones it should look different.

**Comment by microdockman** [comment]
> Was the screenshot from the iPad or pc? At least for phones it should look different.

This is a pc screenshot, of course, ipad screen is quite small, and the preview page will be relatively slow, if triggered hover, sliding up and down will often interrupt, the experience is not very good

**Comment by truefriend-cz** [comment]
I have same problem.
